public/app/plugins/datasource/influxdb/components/ConfigEditor.tsx

```diff
@@ -104,6 +112,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
               isConfigured={(secureJsonFields && secureJsonFields.token) as boolean}
               value={secureJsonData.token || ''}
               label="Token"
+              aria-label="Token"
```

Comment:
It would be nice not to have to define the label twice here. Do you think we could add an `htmlFor` prop to `SecretFormField`?

Reply:
@kaydelaney 
hmm.. in general i agree, but...some comments:
- this pull-request will have to be backported to v8.3.x, so i don't want to make larger changes to other parts, if we can avoid it
- `SecretFormField` is in `@grafana/ui/LegacyForms` ... do we want to update those?
- actually `SecretFormField` does both the label and the input by itself, so we could probably just change it's internals so that it propagates it's label to it's input... if we decide to do it

what do you think?

Reply:
I'm not sure what our policy is re: LegacyForms. (@grafana/plugins-platform-frontend ?)
If it's okay to change, I'd go with what you suggest in your last bullet point

Reply:
@kaydelaney based on what you said, for now i removed the extra `aria-label="Token"` line from this pull-request. this way i think we can merge this, and based on what @grafana/plugins-platform-frontend  is going to say, we are either done, or i can create a single-line-diff PR to bring this back.
do you think this PR is good to be merged, or should i change anything else in it? thanks!